README.md

```diff
@@ -75,7 +74,7 @@ For simplicity, you should check `Enable anonymous access` in the prompts follow
 
 ## Using Cabal With Nexus Repository Manager 3
 
-[We have detailed instructions on how to get started here!](docs/Cabal_USER_DOCUMENTATION.md)
+[We have detailed instructions on how to get started here!](docs/CABAL_USER_DOCUMENTATION.md)
```

Comment:
So now that I've seen you need to fix this more than once, I believe we should just change the filename in the archetype template. We're not married to all CAPS for this name. Also, the template will already have access to an all lower-case name "foo" (format name) and a Capitalized name "Foo" (class name). I think we should just change the archetype to use a simpler file name. Maybe "user_doc....md" could just go all lowercase, and use the "format name" in the template?

Reply:
@bhamail keen eye and great insight! I'll make this corresponding change in my `docs` branch right now. To confirm: `__pluginClass___USER_DOCUMENTATION.md` would be changed to just `__pluginFormat___user_documentation.md`?